addons/material-forum-editor-buttons/addon.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Material forum editor buttons",
+  "description": "Changes the forum editor icons from tiny 16x16 bitmaps to SVG Google Material Icons.",
+  "userstyles": [
+    {
+      "url": "icons.css",
+      "matches": ["https://scratch.mit.edu/discuss/*"]
+    }
+  ],
+  "tags": ["community", "forums", "theme"],
+  "credits": [
+    {
+      "name": "A-E-",
+      "link": "https://github.com/easrng/material-scratch-forum-editor-buttons"
+    },
+    {
+      "name": "Secret-chest (Port)",
+      "link": "https://github.com/Secret-chest"
+    }
+  ],
+  "enabledByDefault": false
```

Comment:
`l10n: true`
```suggestion
  "enabledByDefault": false,
  "l10n": true
```

Reply:
What is that?

Reply:
> What is that?

It allows the addon name and description to be translated.